### DISCUSSION TITLE
Latest version

### DISCUSSION THREAD

**Discussion by ric743256** [leecher1337/ntvdmx64#291] [General]
I hate to ask this dumb question, but I not a programmer and don't know how to compile and am looking for the latest version of ntvdmx64. Can I compile on Windows 11 and if not what can I do.

Rick

**Comment by leecher1337** [comment]
Yes, compiling on Win 11 also works, just follow the instructions in https://github.com/leecher1337/ntvdmx64/blob/master/ntvdmpatch/doc/autobuild.txt
